libcnb-package/src/buildpack_dependency_graph.rs

```diff
@@ -86,7 +85,7 @@ fn build_libcnb_buildpack_dependency_graph_node(
 #[derive(thiserror::Error, Debug)]
 pub enum BuildBuildpackDependencyGraphError {
     #[error("IO error while finding buildpack directories: {0}")]
```

Comment:
I think this error message needs updating, since it's not only IO errors that can occur in this enum variant now.